server/api/db/sqldb/models.py

```diff
@@ -769,6 +774,20 @@ class TimeWindowTracker(Base, mlrun.utils.db.BaseModel):
         def get_identifier_string(self) -> str:
             return f"{self.key}"
 
+    class FunctionSpecFoo(Base, mlrun.utils.db.BaseModel):
+        __tablename__ = "function_spec_foo"
+
+        data = Column(
+            String(length=255, collation=SQLTypesUtil.collation()), primary_key=True
+        )
+        function = relationship(Function, back_populates="foo_spec")
```

Comment:
can you explain what `back_populates` means?

Reply:
back_populates is like using backref, but when using backref the other side of the relationship is created dynamically and is not explicitly written in the code.